source/Connection/AMDSDataStream.cpp

```diff
@@ -329,8 +329,8 @@ void AMDSDataStream::encodeClientRequestType(const AMDSClientRequest &clientRequ
 void AMDSDataStream::write(const AMDSClientRequest &clientRequest){
 	int errorCode = clientRequest.writeToDataStream(this);
 	if (errorCode != AMDS_CLIENTREQUEST_SUCCESS) {
-		QString errorMessage = AMDSClientRequestDefinitions::errorMessage(errorCode, AMDSClientRequestDefinitions::Write, clientRequest.requestType());
-		AMDSErrorMon::alert(0, errorCode, errorMessage);
+		QString errMessage = AMDSClientRequestDefinitions::errorMessage(errorCode, AMDSClientRequestDefinitions::Write, clientRequest.requestType());
```

Comment:
I would prefer it if you did not make your names more terse.  `errorMessage` is preferred to `errMessage`.